eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/ConnectionParameters.java

```diff
@@ -16,48 +16,57 @@
  * A wrapper class that contains all parameters that are needed to establish a connection to an Event Hub.
  */
 public class ConnectionParameters {
-    private final CredentialInfo credentials;
     private final Duration timeout;
-    private final TokenProvider tokenProvider;
+    private final TokenCredential tokenCredential;
     private final TransportType transport;
     private final Retry retryPolicy;
     private final ProxyConfiguration proxyConfiguration;
     private final Scheduler scheduler;
+    private final String host;
+    private final String eventHubPath;
+    private final CBSAuthorizationType authorizationType;
 
-    public ConnectionParameters(final CredentialInfo credentials, final Duration timeout, final TokenProvider tokenProvider,
-                         final TransportType transport, final Retry retryPolicy,
-                         final ProxyConfiguration proxyConfiguration, final Scheduler scheduler) {
-        Objects.requireNonNull(credentials);
+    public ConnectionParameters(String host, String eventHubPath, TokenCredential tokenCredential,
```

Comment:
If this were public API I would be arguing for this to be named 'ConnectionOptions'. Because it isn't I won't, but you might want to consider it to fall in line with the general naming scheme.

Reply:
I can rename this ConnectionOptions.